src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -40,32 +39,64 @@ export default class DiscussionArea {
     this.#lecture = lecture;
     this.#manager = new DiscussionManager(this.#lecture);
     this.#currentTimeMs = 0;
-    this.#currentRootCommentElements = [];
     this.#nearestComments = [];
   }
 
   /**
    * Initialize the discussion area by loading the current comments.
    */
   async initialize() {
-    await this.loadDiscussion();
+    await this.updateDiscussion();
```

Comment:
How is this related to your PR? Do you need to sync?

Reply:
Just merged, should only have the new changes now